.github/workflows/manual-deploy.yaml

```diff
@@ -0,0 +1,36 @@
+name: Manual Deploy
+
+on:
+  workflow_dispatch:
```

Comment:
this workflow will allow us to manually run deploys, selecting environments and which applications to deploy.